src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceDocTransformer.java

```diff
@@ -85,7 +89,23 @@ private ViewModel generateDoc(ProtoFile file, GapicProductConfig productConfig)
             productConfig, ImportSectionView.newBuilder().build(), namer));
     doc.elementDocs(elementDocTransformer.generateElementDocs(typeTable, namer, file));
     doc.modules(
-        generateModuleViews(file.getModel(), productConfig, namer, isSourceApiInterfaceFile(file)));
+        generateModuleViews(
+            file.getModel(), productConfig, namer, isSourceApiInterfaceFile(file), false));
+    return doc.build();
+  }
+
+  private ViewModel generateOverview(Model model, GapicProductConfig productConfig) {
+    // TODO(jgeiger): switch to use productConfig.getPackageName() instead of
```

Comment:
This namer already uses `productConfig.getPackageName()`

Reply:
Removed.